### ISSUE TITLE
Drawn MIDI curves are steppy on hardware syntzs

### ISSUE THREAD

**Issue by MegaloMIDIac** [SynthstromAudible/DelugeFirmware#501]
### Please describe the problem:

If you draw a MIDI curve with the pads on a MIDI CC, it will sound steppy. If you record MIDI CC from the external synth all is fine. 

### What is the expected behavior?

Smooth transitions 

### What hardware did you reproduce it with?

7SEG

### What firmware did you reproduce it with?

nightly

**Comment by m-m-adams** [comment]
Can you enable interpolation on the external synth? 

Interpolating all midi to a high resolution on the deluge would significantly increase the amount of CC data sent which 
1) impacts note on/off timing since it's a serial protocol
2) requires more cpu time in the deluge

The recording works fine because your synth is sending CCs at a higher rate than the deluge automation currently generates them (1 per step at current zoom level). We can play with that frequency a little but without interpolating externally it will always be a bit steppy

**Comment by MegaloMIDIac** [comment]
I have tested this behaviour with the Microfreak, Ambika and Blofeld. I don't think they can interpolate incoming MIDI CC.
You can hear this especially if you use the filter cutoff CC.

**Comment by seangoodvibes** [comment]
This is with interpolation on in automation view right?

How are you inputting the MIDI automation lanes on the deluge? 

If you are changing one step at a time, it will be stepped regardless of whether interpolation is on or off. 

But if you enter a long press (press two pads to connect them) it will be interpolated when interpolation is on and stepped when interpolation is off.

**Comment by MegaloMIDIac** [comment]
I tried both, with and without interpolarion.
Also the long press will get you the steppy result unfortunately.

**Comment by seangoodvibes** [comment]
Thanks for reporting this @MegaloMIDIac 

I just confirmed the same behaviour sending MIDI to my dirtywave m8.

Hopefully we can find a solution for this as part of our first release as this is a notable bug.

Cheers

Sean

**Comment by m-m-adams** [comment]
Yeah the automation view sets a value for each pad at the current zoom level. Properly smoothing to only change cc value by 1 at a time will require deeper changes and still sound steppy on many synths
